src/pstress.cpp

```diff
@@ -49,6 +52,20 @@ void create_worker(struct workerParams *Params) {
 }
 
 int main(int argc, char *argv[]) {
+
+  /* Extract the fullpath of the executable binary */
+  char buffer[100];
```

Comment:
The size `100` would be too less if pstress is located deep in the directory structure. May be it is better to change it to 512 as mysqld also uses 512 for buffers for storing absolute paths (See `FN_REFLEN`).

Suggestion:
```C
#define PATH_MAX 512
```
and use it as 
```C
char buffer[PATH_MAX];
```

Reply:
fixed